cli/internal/cmd/miniup.go

```diff
@@ -143,15 +143,16 @@ func (m *miniUpCmd) prepareConfig(cmd *cobra.Command, fileHandler file.Handler,
 		}
 	}
 
-	config := config.Default()
-	config.Name = constants.MiniConstellationUID
-	config.RemoveProviderAndAttestationExcept(cloudprovider.QEMU)
-	config.StateDiskSizeGB = 8
-
+	// TODO: Should we allow this at all?  It seems more logical to follow the usage pattern for the other providers: constellation config generate mini -> constellation init. But maybe it's more user friendly to not persist a config file in the workspace for a testing cluster?
+	config := config.MiniDefault()
```

Comment:
The mini constellation command was created to explicitly skip the whole config-generate -> create -> init flow and provide one single command to handle all of this.